streams/src/test/java/org/apache/kafka/streams/processor/internals/ActiveTaskCreatorTest.java

```diff
@@ -173,9 +200,24 @@ public void shouldThrowStreamsExceptionOnErrorCloseThreadProducerIfEosDisabled()
     public void shouldCreateProducerPerTaskIfEosAlphaEnabled() {
         properties.put(StreamsConfig.PROCESSING_GUARANTEE_CONFIG, StreamsConfig.EXACTLY_ONCE);
         mockClientSupplier.setApplicationIdForProducer("appId");
+
+        final Map<String, Object> mockProducerConfig = mock(Map.class);
+        expect(mockProducerConfig.put(ProducerConfig.TRANSACTIONAL_ID_CONFIG, "appId-0_0")).andReturn(null);
+        expect(mockProducerConfig.put(ProducerConfig.TRANSACTIONAL_ID_CONFIG, "appId-0_1")).andReturn(null);
```

Comment:
For eos-alpha, we set the `transcational.id` for each task

Reply:
Good call, thanks for the improved coverage.